client/editSchedule.js

```diff
@@ -190,46 +158,31 @@ Template.addEditSomethingModal.helpers({
 });
 
 AutoForm.hooks({
-  editRoundForm: {
+  editEventForm: {
     onSubmit: function(insertDoc, updateDoc, currentDoc) {
       this.event.preventDefault();
 
       if(currentDoc._id) {
-        // Updating an existing round
-        Rounds.update({
-          _id: currentDoc._id,
-        }, {
-          $set: {
-            title: insertDoc.title,
-            startMinutes: insertDoc.startMinutes,
-            durationMinutes: insertDoc.durationMinutes,
-          }
-        });
-        $('#addEditSomethingModal').modal('hide');
+        delete updateDoc.$set.competitionId; // Don't touch the competitionId field. We don't trust browser input that much.
```

Comment:
I always get uneasy when I see code that looks like it's maybe meant to be for security and it's running client side. What is going on here? Over on the server, it looks like we don't allow the `competitionId` field to be changed: https://github.com/larspetrus/ccm/blob/master/both/access_control.js#L240.